creedengopython.properties

```diff
@@ -1,12 +1,19 @@
 category=External Analysers
 description=Provide a list of static code analyzers (for Python language) to highlight code structures that may have a negative ecological impact: energy and resources over-consumption, "fatware", shortening terminals' lifespan, etc.
 homepageUrl=https://github.com/green-code-initiative/creedengo-python
-publicVersions=2.0.0
-archivedVersions=
+publicVersions=2.0.1
+archivedVersions=2.0.0
 
 defaults.mavenGroupId=org.green-code-initiative
 defaults.mavenArtifactId=creedengo-python-plugin
 
+2.0.1.description=improve library versions; check and upgrades for SonarQube until 25.3.0 version
+2.0.1.sqcb=[24.12,25.3.*]
+2.0.1.sqVersions=[9.9,10.7.*]
+2.0.1.date=2025-03-14
+2.0.1.downloadUrl=https://github.com/green-code-initiative/creedengo-python/releases/download/2.0.1/creedengo-python-plugin-2.0.1.jar
+2.0.1.changelogUrl=https://github.com/green-code-initiative/creedengo-python/releases/tag/2.0.1
+
 2.0.0.description=Initial Marketplace publication of plugin after renaming from "ecocode-python"
 2.0.0.sqs=[10.8,LATEST]
```

Comment:
An archived version can't be compatible with `LATEST`. You should update the compatibility of 2.0.0 to end the ranges with the latest relevant versions.

Reply:
ok, thank you, done.